src/core/radio/trel_link.cpp

```diff
@@ -373,8 +376,42 @@ void Link::ProcessReceivedPacket(Packet &aPacket)
 
     Get<Mac::Mac>().HandleReceivedFrame(&mRxFrame, kErrorNone);
 
+    // As the received frame is processed by the MAC or MLE layers,
```

Comment:
shouldn't these comments be above `HandleReceivedFrame` ? As I understand, first `mRxPacketPeer` is set above, then `HandleReceivedFrame()` is called which might make use of the `mRxPacketPeer` information, and then after this `mRxPacketPeer` is always set to `null` in the return below.

So that would be more clear if the comments are above it.

Reply:
Thanks. Moved the comment above the `HandleReceivedFrame()`.